From b0f9ee06c9e611592715f9d9d31170d55d2aeded Mon Sep 17 00:00:00 2001
From: Dave Stevenson <dave.stevenson@raspberrypi.org>
Date: Thu, 16 May 2019 17:49:42 +0100
Subject: [PATCH] drm: vc4-firmware-kms: Fix DSI display support

The mode was incorrectly listed as interlaced, which was then
rejected.
Correct this and FKMS works with the DSI display.

Signed-off-by: Dave Stevenson <dave.stevenson@raspberrypi.org>
---
 drivers/gpu/drm/vc4/vc4_firmware_kms.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/drivers/gpu/drm/vc4/vc4_firmware_kms.c
+++ b/drivers/gpu/drm/vc4/vc4_firmware_kms.c
@@ -1005,7 +1005,7 @@ static const struct drm_display_mode lcd
 		 25979400 / 1000,
 		 800, 800 + 1, 800 + 1 + 2, 800 + 1 + 2 + 46, 0,
 		 480, 480 + 7, 480 + 7 + 2, 480 + 7 + 2 + 21, 0,
-		 DRM_MODE_FLAG_INTERLACE)
+		 0)
 };
 
 static int vc4_fkms_lcd_connector_get_modes(struct drm_connector *connector)
